public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -10,6 +10,7 @@ import {
   VizPanelMenu,
   VizPanelState,
 } from '@grafana/scenes';
+import { LibraryPanel } from '@grafana/schema/dist/esm/index.gen';
```

Comment:
Ain't this exported directly from the schema package? THink we should not import from the dist path.

Reply:
If not then, I guess we need to use the depraecated type, omg

Reply:
Yeah, sorry. VS code auto importing :)